.github/scripts/bake-metadata.py

```diff
@@ -136,9 +146,13 @@ def get_tags(self, target: Target) -> List[str]:
         return [*super().get_tags(target), self.pr_tag(target)]
 
     def get_cache_to(self, target: Target) -> List[str]:
+        if self.is_fork:
+            return []
         return [registry_cache(self.pr_tag(target))]
 
     def get_cache_from(self, target: Target) -> List[str]:
+        if self.is_fork:
+            return []
```

Comment:
We should probably load from the target branch cache here to avoid starting from scratch.

See hack commit here, brings build time from 30min down to 10min: https://github.com/OpenRailAssociation/osrd/commit/3a14f328cc2283343a5d6d6b225b36de393769bb

Reply:
Yup nice one